### ISSUE TITLE
Custom fonts didn't work on Android 7

### ISSUE THREAD

**Issue by jessejiang0214** [xamarin/xamarin-android#1376]
### Steps to Reproduce

1. Follow this  [document](https://developer.xamarin.com/guides/android/application_fundamentals/resources_in_android/fonts/) to use a custom font as a resources
2. Deploy on Android 7
3. 

### Expected Behavior
1. It should display the custom font on Android 7.
2. The document said we need use this line in TextView 
`app:fontFamily="@font/caveat_bold"`
### Actual Behavior
1. It can display the custom font on Android 8, but not Android 7.
2. I got `No resource identifier found for attribute 'fontFamily' in package 'com.xamconsulting.TestFont'` error, when I add `app:fontFamily="@font/caveat_bold"` into TextView
### Log File

=== Visual Studio Community 2017 for Mac ===

Version 7.4 (build 1033)
Installation UUID: 57bfe2c6-725d-410d-b5f2-52b69d97c377
Runtime:
	Mono 5.8.0.129 (2017-10/835a98d8559) (64-bit)
	GTK+ 2.24.23 (Raleigh theme)

	Package version: 508000129

=== NuGet ===

Version: 4.3.1.4445

=== .NET Core ===

Runtime: /usr/local/share/dotnet/dotnet
Runtime Versions:
	2.0.5
	2.0.0
	1.1.1
	1.0.8
	1.0.4
SDK: /usr/local/share/dotnet/sdk/2.1.4/Sdks
SDK Versions:
	2.1.4
	2.0.0
	1.0.1
MSBuild SDKs: /Library/Frameworks/Mono.framework/Versions/5.8.0/lib/mono/msbuild/15.0/bin/Sdks

=== Xamarin.Profiler ===

'/Applications/Xamarin Profiler.app' not found

=== Xamarin.Android ===

Version: 8.2.0.15 (Visual Studio Community)
Android SDK: /Users/jesse/Library/Developer/Xamarin/android-sdk-macosx
	Supported Android versions:
		6.0 (API level 23)
		7.0 (API level 24)
		7.1 (API level 25)
		8.0 (API level 26)

SDK Tools Version: 26.1.1
SDK Platform Tools Version: 27.0.1
SDK Build Tools Version: 26.0.3

Java SDK: /usr
java version "1.8.0_131"
Java(TM) SE Runtime Environment (build 1.8.0_131-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.131-b11, mixed mode)

Android Designer EPL code available here:
https://github.com/xamarin/AndroidDesigner.EPL

=== Xamarin Inspector ===

Version: 1.4.0
Hash: b3f92f9
Branch: master
Build date: Fri, 19 Jan 2018 22:00:34 GMT
Client compatibility: 1

=== Apple Developer Tools ===

Xcode 9.2 (13772)
Build 9C40b

=== Xamarin.Mac ===

Version: 4.2.0.20 (Visual Studio Community)

=== Xamarin.iOS ===

Version: 11.8.0.20 (Visual Studio Community)
Hash: 1c6f300e
Branch: d15-6
Build date: 2018-02-07 23:42:40-0500

=== Build Information ===

Release ID: 704001033
Git revision: 053991068f3c88497d3e5cc52fe7fdf57c28b62f
Build date: 2018-02-28 05:02:28-05
Xamarin addins: cc3283953e0af0b8bfab1bdfdf847d9e403c4606
Build lane: monodevelop-lion-d15-6

=== Operating System ===

Mac OS X 10.13.3
Darwin 17.4.0 Darwin Kernel Version 17.4.0
    Sun Dec 17 09:19:54 PST 2017
    root:xnu-4570.41.2~1/RELEASE_X86_64 x86_64

=== Enabled user installed addins ===

NuGet Package Management Extensions 0.12.2
Internet of Things (IoT) development (Preview) 7.1

[Demo project](https://github.com/xamarin/xamarin-android/files/1787631/TestFont.zip)

**Comment by jessejiang0214** [comment]
[TestFont.zip](https://github.com/xamarin/xamarin-android/files/1787631/TestFont.zip)

**Comment by atsushieno** [comment]
Custom fonts are documented as supported only after Android 8. https://developer.android.com/guide/topics/ui/look-and-feel/fonts-in-xml.html

**Comment by jessejiang0214** [comment]
@atsushieno 

> The Support Library 26.0 provides support to the Fonts in XML feature on devices running Android 4.1 (API level 16) and higher.

**Comment by atsushieno** [comment]
Then it's not about xamarin-android anymore -> https://github.com/xamarin/AndroidSupportComponents /cc @Redth

**Comment by atsushieno** [comment]
Actually I think it's more about tooling capabilities issue, which comes back to xamarin-android issue. The Gradle build seems to do some trick with `tools:overrideLibrary` attribute, which is (1) currently not merged into the app AndroidManifest.xml (our manifest merger support is limited) and (2) there is no way too hook Gradle extra tasks into our build process (I have no idea what Android Gradle Plugin does there; it might not be doing anything special). But in any case, adding `<uses-sdk>` extra element in the app directly did not help.

https://developer.android.com/studio/build/manifest-merge.html#override_wzxhzdk38uses-sdk_for_imported_libraries

**Comment by atsushieno** [comment]
Nevermind, manifest-merger and gradle processing had nothing to do with this. When you use support library version of custom fonts, you have to use Activity from support library. Your sample code worked if I switched to AppCompatActivity.

**Comment by jessejiang0214** [comment]
@atsushieno Thanks so much, you solve my problems.
